bears/r/FormatRBear.py

```diff
@@ -14,8 +19,51 @@ class FormatRBear:
     LANGUAGES = "R"
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
+    def create_arguments(filename, file, config_file,
+                         r_keep_comments: bool=True,
+                         r_keep_blank_lines: bool=True,
+                         r_braces_on_next_line: bool=False,
+                         r_use_arrows: bool=False,
+                         tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH):
+        """
+        :param r_keep_comments:
+            Determines whether comments are kept or not.
+        :param r_keep_blank_lines:
+            Determines whether blank lines are kept or not.
+        :param r_braces_on_next_line:
+            Determines whether a brace should be placed on the next line.
+            Example: If ``True`` then
+            ```
+            if (...) {
+            ```
+            changes to
+            ```
+            if (...)
+            {
+            ```
+            If ``False`` the brace is placed on the same line.
+        :param r_use_arrows:
+            Determines if either the assign operator ``=`` or the arrow ``<-``
+            should be used.
+            Example:
+            If ``True``, ``x = 1`` changes to ``x <- 1``.
+            If ``False``, ``x <- 1`` changes to ``x = 1``.
+        :param tab_width:
+            Number of spaces for indentation.
+        """
         rcode = ('library(formatR);'
-                 'formatR::tidy_source(source="' +
-                 escape(filename, '"\\') + '")')
+                 'formatR::tidy_source('
+                 'source="' + escape(filename, '"\\') + '",'
+                 'comment={r_keep_comments},'
+                 'blank={r_keep_blank_lines},'
+                 'arrow={r_use_arrows},'
+                 'brace.newline={r_braces_on_next_line},'
+                 'indent={tab_width}'
+                 ')'.format(tab_width=tab_width,
+                            r_keep_comments=_map_to_r_bool(r_keep_comments),
+                            r_keep_blank_lines=_map_to_r_bool(
+                                r_keep_blank_lines),
+                            r_use_arrows=_map_to_r_bool(r_use_arrows),
+                            r_braces_on_next_line=_map_to_r_bool(
+                                r_braces_on_next_line)))
```

Comment:
just generally: checks can be disabled by not having used the parameters inside `tidy_source`, right? If so we should file an issue after merge to allow users to omit checks if not wanted :+1: